packages/select/index.ts

```diff
@@ -20,46 +20,57 @@ import { messages as svMessages } from './locales/sv/messages.mjs';
 
 import '@warp-ds/icons/elements/chevron-down-16';
 
-export class WarpSelect extends FormControlMixin(kebabCaseAttributes(WarpElement)) {
-  static properties = {
-    // Whether the element should receive focus on render
-    autoFocus: { type: Boolean, reflect: true },
+export class WarpSelect extends FormControlMixin(LitElement) {
+  // Whether the element should receive focus on render
+  @property({ type: Boolean, reflect: true })
+  autoFocus: boolean;
 
-    // Renders the field in an invalid state. Often paired with `hint` to provide feedback about the error
-    invalid: { type: Boolean, reflect: true },
+  // Renders the field in an invalid state. Often paired with `hint` to provide feedback about the error
+  @property({ type: Boolean, reflect: true })
+  invalid: boolean;
 
-    // Whether to always show a hint
-    always: { type: Boolean, reflect: true },
+  // Whether to always show a hint
+  @property({ type: Boolean, reflect: true })
+  always: boolean;
```

Comment:
I think the name is a bit vague

```suggestion
  showHint: boolean;
```

Reply:
No breaking changes this round